package/MDAnalysis/core/universe.py

```diff
@@ -191,6 +191,9 @@ class Universe(object):
         representations, which allow for manipulation of coordinates.
     in_memory_step
         Only read every nth frame into in-memory representation.
+    continuous : bool, optional
```

Comment:
This would benefit from some more details in the description part of the docstring. The description in the PR message (with the ascii art) should do.

Reply:
Or at least a link to the ChainReader.